test/api.test.js

```diff
@@ -1,5 +1,15 @@
-import test from 'ava'
+const newman = require('newman')
 
-test('foo', t => {
-  t.pass()
-})
+newman.run(
+  {
+    collection: require('../docs/agent.postman_collection.json'),
+    environment: require('../docs/agent.postman_environment.json'),
+    reporters: 'cli'
```

Comment:
So CodeClimate, as with pretty much every js project, requires reports in `lcov` format but Newman seems to not be able to produce that. So would just try and see if the ones supported by newman work too:

```suggestion
    reporters: ['cli', 'json', 'junit']
```